.circleci/config.yml

```diff
@@ -137,7 +160,7 @@ parameters:
   linux-machine-image:
     type: string
     # Latest supported ubuntu image from https://circleci.com/docs/2.0/configuration-reference/#available-machine-images
-    default: "ubuntu-2004:202010-01"
+    default: "ubuntu-2004:2022.10.1"
```

Comment:
I saw we were using a pretty old image so I updated it to the last one. I haven't seen any issues with the update.